docs/content.zh/docs/dev/table/sql-gateway/overview.md

```diff
@@ -116,6 +116,34 @@ $ curl --request GET http://localhost:8083/v1/sessions/${sessionHandle}/operatio
 
 结果中的 `nextResultUri` 不是null时，用于获取下一批结果。
 
+### Deploy Script
+
+SQL Gateway supports to deploy a script in [Application Mode]({{< ref "docs/deployment/overview" >}}). In application mode, Job Master is responsible for the script compiling.
```

Comment:
nit: to deploy -> deploying
what do we mean by Job Master, we should have a link - do you mean the JobManager?
the script compiling. -> compiling the script.